modules/n1ql/pages/n1ql-language-reference/createsequence.adoc

```diff
@@ -0,0 +1,412 @@
+= CREATE SEQUENCE
+:page-topic-type: reference
+:page-status: Couchbase Server 7.6
+:imagesdir: ../../assets/images
+:description: The CREATE SEQUENCE statement enables you to create a sequence in a given keyspace.
+
+// Cross-references
+:authorization-overview: xref:learn:security/authorization-overview.adoc
+:query-context: xref:n1ql:n1ql-intro/queriesandresults.adoc#query-context
+:identifiers: xref:n1ql-language-reference/identifiers.adoc
+:logical-hierarchy: xref:n1ql-intro/sysinfo.adoc#logical-hierarchy
+
+// TEMP
+include::partial$n1ql-language-reference/horizontal-style.adoc[]
+
+[abstract]
+{description}
+
+== Purpose
+
+include::./sequenceops.adoc[tags=overview;attributes]
+
+=== Cache
+
+Sequences operate most efficiently with an in-memory cache of values.
+You can specify the size of this cache when you create the sequence.
+A block of values is reserved by a node, and requests for values are satisfied from this cache.
+When exhausted, a new block of values is reserved.
+Reserving a cached block offers a performance boost, as it enables the Query service to return values directly from memory.
+
+Note however that if a Query node shuts down, or if you alter the sequence, the unused values in the cached block are lost: a new block is reserved when you restart the node, or request the next value.
+You should choose a cache size with this in mind, along with the expected usage patterns for the sequence.
+
+=== Storage
+
+Sequences are stored in the bucket's hidden `&lowbar;system` scope.
+When you back up or restore a bucket, sequences are included automatically.
```

Comment:
It may not be necessary to note this; sequence are restored in accordance with the restore command - that is, if select scopes are restored, so are the sequences belonging to those scopes (but not others).  That is, the restore granularity is possible at a scope level.  (As with everything else, backup's filters are used when processing.)

Reply:
Done.